files/es/web/css/css_container_queries/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: CSS Container Queries
+slug: Web/CSS/CSS_Container_Queries
+page-type: guide
+tags:
+  - CSS
+  - CSS Containment
+  - CSS Container Queries
+  - Guide
+  - Paint
+  - Responsive Design
+l10n:
+  sourceCommit: 674fd4fb4225dfad82eab7425f9fc8e52d67a7db
```

Comment:
-  Delete duplicated tags
- Using git and looking at the `en-US` file in the repo I got this commit, could you send me how did you obtain that one?

```suggestion
l10n:
  sourceCommit: 43edd92d6fe6da31cd5ee0874b888244826fb1f7
```

Reply:
Ahora que hago memoria seguramente tomé el último commit de mdn/translated-content antes de que empezara a traducir, pero en realidad debía tomar un commit de mdn/content :face_with_spiral_eyes:

Reply:
Ya veo, por eso salen diferentes 😅, muchas gracias! entonces para aplicar el cambio y hacer el merge

Muchas gracias!